editoast/src/client/mod.rs

```diff
@@ -20,6 +20,7 @@ pub struct Client {
 pub enum Commands {
     Runserver(RunserverArgs),
     Generate(GenerateArgs),
+    Import(ImportArgs),
```

Comment:
To better match the issue:

```suggestion
    ImportRailjson(ImportArgs),
```